src/java.base/share/classes/java/math/BigInteger.java

```diff
@@ -603,74 +603,55 @@ public BigInteger(String val, int radix) {
     }
 
     /*
-     * Constructs a new BigInteger using a char array with radix=10.
-     * Sign is precalculated outside and not allowed in the val. The {@code val}
-     * array is assumed to be unchanged for the duration of the constructor
```

Comment:
This removed comment about the array being assumed to be unchanged is still applicable to the `CharSequence` (since it'll be backed by the passed in `char[]`).